src/memray/_memray/inject.cpp

```diff
@@ -0,0 +1,245 @@
+#define Py_LIMITED_API 0x03070000
+#include "Python.h"
+
+#include <netdb.h>
+#include <netinet/in.h>
```

Comment:
Are these needed? I can compile in Linux without the `in.h` and `tcp.h`

Reply:
Hm, maybe not? I'm sure I copy-pasted those from some man page, but I don't see which now... Looks like POSIX says `netinet/tcp.h` only provides `TCP_NODELAY`, which we definitely don't use. `netinet/in.h` provides quite a lot more. It looks like we would need it if we were calling `setsockopt`, but we're not...

I think you're right, I don't think we're making use of anything from either of them. I'll drop them.